installing/installing_restricted_networks/installing-restricted-networks-preparations.adoc

```diff
@@ -5,38 +5,93 @@ include::modules/common-attributes.adoc[]
 
-Before you create the mirror registry, you must prepare the bastion host.
+Before you perform the mirror procedure, you must prepare the host to retrieve content
+and push it to the remote location.
 
 include::modules/cli-installing-cli.adoc[leveloffset=+2]
 
-include::modules/installation-creating-mirror-registry.adoc[leveloffset=+1]
+include::modules/installation-adding-mirror-registry-pull-secret.adoc[leveloffset=+1]
 
-include::modules/installation-local-registry-pull-secret.adoc[leveloffset=+1]
+[IMPORTANT]
+===
+This step requires that you have write access to a container image repository on the
+mirror registry and adding those credentials to a registry pull secret. This secret
+should NOT be used to install a cluster as it would give machines within the cluster
+write access to your mirror registry.
+===
+
+[id="installing-performing-connected-mirror"]
+== Performing a mirror while connected to the internet
 
-//include::modules/installation-adding-registry-pull-secret.adoc[leveloffset=+1]
+$ oc adm release mirror OPENSHIFT_VERSION --to MIRROR_REPOSITORY
 
 include::modules/installation-mirror-repository.adoc[leveloffset=+1]
 
 include::modules/installation-restricted-network-samples.adoc[leveloffset=+1]
 
+[id="installing-performing-disconnected-mirror"]
+== Performing a mirror to disk for use in disconnected environments
+
+When mirroring to disk you will be downloading images as files, moving your
+mirror host (which may be a laptop) or a removable media device (such as a
+USB drive) into the restricted network, and then completing the mirror
+procedure.
+
+$ oc adm release mirror OPENSHIFT_VERSION --to MIRROR_REPOSITORY --dry-run
+# copy the ICS/ICSP
+
+$ mkdir MIRROR_DIR
+$ cd MIRROR_DIR
+$ oc adm release mirror OPENSHIFT_VERSION --to file://openshift/release
+
+# disconnect, move across firewall
+
+$ oc adm release mirror file://openshift/release:OPENSHIFT_VERSION --to MIRROR_REPOSITORY
+
+[id="installing-performing-disconnected-mirror-without-registry"]
+== Performing a mirror to disk for use in disconnected environments with a non-production mirror registry
+
+If a production mirror registry is not available, you may configure a simple mirror
+registry using the disconnected procedure to serve container images downloaded to disk.
+
+# Determine the IP of your host within the restricted network as <private_ip>
+
+$ oc adm release mirror OPENSHIFT_VERSION --to <private_ip>:5000/openshift/release --dry-run
+# copy the ICS/ICSP for use later
+
+$ mkdir MIRROR_DIR
+$ cd MIRROR_DIR
+$ oc adm release mirror OPENSHIFT_VERSION --to file://openshift/release
+
+# disconnect, move across firewall
+
+# start an image mirror server on port 5000 on all interfaces on the host
+$ cd MIRROR_DIR
+$ oc image serve
+
+This registry will perform no authentication and does not require TLS in order to guarantee
+integrity of the provided images.
+
 .Next steps
 
+# TODO need to add the registry secret to the machines, which is different
```

Comment:
@smarterclayton, do you have the process to add this secret to the machines or a contact who would?